e2e/client.go

```diff
@@ -0,0 +1,24 @@
+// +build e2e
```

Comment:
Here you start with `+` in the readme it's `build + e2e`

Reply:
Yes, this one is ok (== verified on ci :)

Reply:
FYI it's k8srequired now.